src/api/app/controllers/webui2/interconnects_controller.rb

```diff
@@ -0,0 +1,5 @@
+module Webui2::IntrerconnectsController
```

Comment:
Typo: module Webui2::Int<b><strike>r</strike></b>erconnectsController (so removing the extra `r`)